Update charter.txt

BMHebert opened the pull request:
Reordered the sections of the charter and organized it so that it all has a relatively consistent formatting. Added in the section listing the meeting times and created areas for Goals and Work Norms to go. Also did some minor spellchecking and grammatical changes. Signed my name.

BMHebert committed:
Update charter.txt

Reordered the sections of the charter and organized it so that it all has a relatively consistent formatting. Added in the section listing the meeting times and created areas for Goals and Work Norms to go. Also did some minor spellchecking and grammatical changes. Signed my name.

BMHebert committed:
Update charter.txt

Added my contact info

BMHebert committed:
Update charter.txt

Added in Work Norms and Goals sections.

BMHebert commented:
Added in Goals and Work Norms sections since these had apparently been written already and just not merged into the main branch.

bruggin9 commented:
I apologize that was me who created the Goals and Work Norms, but I did not know how to merge those changes to the main branch. Thank you for doing that, I will look into how to do that so it doesn't happen again.

bruggin9 committed:
Update charter.txt

I added my contact information to the document and signed at the bottom. All of the formatting changes looked good.

cogii reviewed (commented):
The formatting looks excellent!
